playbooks/bgp_l3_fabric/clos-fabric.yaml

```
@@ -1,43 +1,42 @@
 ---
-- hosts: datacenter
+- name: "Configure BGP clos fabric options after configuring prerequisite options"
+  hosts: datacenter
+  gather_facts: false
   connection: httpapi
-  gather_facts: no
-  collections:
-     - dellemc.enterprise_sonic
   tasks:
-  - name: Create Interfaces
-    sonic_interfaces:
-      config: "{{sonic_interfaces}}"
-    when: sonic_interfaces is defined and sonic_interfaces
+    - name: Create Interfaces
+      dellemc.enterprise_sonic.sonic.sonic_interfaces:
```

Is this final "sonic" necessary in the path? It wasn't in the collections value that was removed.
Have you ran this playbook to test this path works for all the below modules?

```suggestion
      dellemc.enterprise_sonic.sonic_interfaces:
```

---

I have made the change.